In [ ]:
import pandas as pd
import matplotlib.pyplot as plt
%matplotlib inline

df1 = pd.read_pickle('namus_html1.pkl')
df2 = pd.read_pickle('namus_html2.pkl')

dfs = [df1, df2]
namus = pd.concat(dfs)
print(namus.shape)

In [14]:
zeroStars = namus[namus.case_rating == 'Extremely Low']
zeroStars = zeroStars[['case_number','date_found', 'county', 'state', 'recognizable_face', 'race', 'sex', 'weight', 'height', 'est_age']]

In [15]:
zeroStars.shape

(526, 10)

In [22]:
zeroStars[0:50]

case_number                date_found      county       state  \
33          06-0457      April 19, 2006 00:00  Palm Beach     Florida   
222         02-1510    October 06, 2002 00:00      Fulton     Georgia   
226       1445-93-A    October 10, 1993 12:00     Boulder    Colorado   
316         95-2360   December 13, 1995 00:00      Fulton     Georgia   
333         93-0576      March 18, 1993 00:00      Fulton     Georgia   
353         89-1991    October 18, 1989 00:00      Fulton     Georgia   
374  1E00 / 00-0036    January 09, 2000 00:00       Duval     Florida   
392         ME G-49  September 19, 1973 14:20  Palm Beach     Florida   
394        ME 70-80       June 23, 1980 14:42  Palm Beach     Florida   
395     ME 82-P-203       July 14, 1982 09:44  Palm Beach     Florida   
419        82-P-204       July 14, 1982 11:26  Palm Beach     Florida   
435         87-0477        May 07, 1987 10:30  Palm Beach     Florida   
553      2005-01240   November 11, 2005 00:00  Palm Beach     Florida   
557       2004-1202   November 20, 2004 00:00  Palm Beach     Florida   
558        04/08/04     August 05, 2004 00:00  Palm Beach     Florida   
559        04/06/04       June 22, 2004 00:00  Palm Beach     Florida   
564      2003-00977  September 07, 2003 00:00  Palm Beach     Florida   
572      2002-00710     August 01, 2002 00:00  Palm Beach     Florida   
578      2001-DA-02   November 23, 2001 00:00      DeSoto     Florida   
585      2002-00015    January 06, 2002 00:00   Charlotte     Florida   
587      2000-00921   November 21, 2000 00:00  Palm Beach     Florida   
588      2000-00922   November 18, 2000 00:00  Palm Beach     Florida   
594        99-20149   December 14, 1999 00:00      Martin     Florida   
597      1999-00579       July 23, 1999 00:00  Palm Beach     Florida   
598      1999-00427        May 23, 1999 00:00  Palm Beach     Florida   
602      1999-00282      March 27, 1999 00:00  Palm Beach     Florida   
614      1996-01005   November 27, 1996 00:00  Palm Beach     Florida   
620      1995-00830    October 05, 1995 00:00  Palm Beach     Florida   
623      1995-00281        May 06, 1995 00:00   St. Lucie     Florida   
633       1993-0867    October 25, 1993 00:00  Palm Beach     Florida   
641      1992-92-07     August 11, 1992 00:00  Palm Beach     Florida   
650       1991-0456        May 17, 1991 00:00  Palm Beach     Florida   
652       1991-0415        May 04, 1991 00:00  Palm Beach     Florida   
661      1990-00232      March 23, 1990 00:00         Lee     Florida   
663      1990-00006    January 04, 1990 00:00         Lee     Florida   
666       1989-1302     August 09, 1989 00:00     Broward     Florida   
674      1989-00001    January 01, 1989 00:00         Lee     Florida   
679      1988-00171     August 23, 1988 00:00     Collier     Florida   
682      1988-00858       July 23, 1988 00:00       Duval     Florida   
711          NPD322       June 07, 1988 00:00       Essex  New Jersey   
714         7852154     August 20, 1985 00:00       Essex  New Jersey   
716         7853216   December 15, 1985 00:00       Essex  New Jersey   
718         7852728    October 24, 1985 00:00       Essex  New Jersey   
720         8632125      April 25, 1986 00:00       Essex  New Jersey   
722         7862292  September 21, 1986 00:00       Essex  New Jersey   
760         00-017H   February 09, 2000 00:00       Essex  New Jersey   
774       MCU920118      April 27, 1992 00:00  Gloucester  New Jersey   
848          150-86   November 28, 1986 00:00       Essex  New Jersey   
849           44-91        May 10, 1991 00:00       Essex  New Jersey   
850        98-92369    October 13, 1998 00:00       Essex  New Jersey   

                                     recognizable_face  \
33         Not recognizable - Decomposing/putrefaction   
222                                                      
226                                                      
316                                                 

* **Index 374 Not recognizable, Cannot Estimate, Estimated**
* **Index 435 Not recognizable, 148, 71 (No text)** 
* **Index 650 Not recognizable, Estimated height and weight**
* **Index 666 Not recognizable, Cannot estimate, Estimated**
* **Index 679 Not recognizable, Estimated height and weight**
* **Index 682 Not recognizable, Estimated height and weight**
* **Index 720 Not recognizable, Estimated height and weight, Cannot Determine est_age**

In [23]:
zeroStars[50:100]

case_number                date_found       county       state  \
864        9607534  September 06, 1996 00:00        Essex  New Jersey   
873        I86-398   December 04, 1986 00:00   Gloucester  New Jersey   
875        00-028H      March 08, 2000 00:00        Essex  New Jersey   
903        8805215        May 01, 1988 00:00       Hudson  New Jersey   
924       94-46053  September 08, 1994 00:00    Middlesex  New Jersey   
925        9543041     August 18, 1995 00:00    Middlesex  New Jersey   
930        89-5160       June 10, 1989 00:00      Passaic  New Jersey   
994        2426-03       June 25, 2003 17:55       Dallas       Texas   
995      JP0916-03      March 06, 2003 18:20      Navarro       Texas   
999        4619-02   December 12, 2002 16:39       Dallas       Texas   
1004       1438-02      April 14, 2002 16:57       Dallas       Texas   
1022     JP2841-01       July 29, 2001 10:52        Gregg       Texas   
1084       87-2054  September 23, 1987 00:00     Maricopa     Arizona   
1093       91-2622   December 07, 1991 00:00     Maricopa     Arizona   
1101       96-0604   February 27, 1996 00:00     Maricopa     Arizona   
1102       96-1454        May 27, 1996 00:00     Maricopa     Arizona   
1127       00-3238   November 02, 2000 12:30     Maricopa     Arizona   
1135      A98-3798  September 12, 1998 00:00     Richmond     Georgia   
1188       04-0406   February 01, 2004 00:00     Maricopa     Arizona   
1191       95-2315     August 15, 1995 00:00     Maricopa     Arizona   
1207      S94-1663      March 15, 1994 11:18      Chatham     Georgia   
1219  2002-1012483      April 06, 2002 09:41      Clayton     Georgia   
1270      C88-2283     August 21, 1988 15:45      Stewart     Georgia   
1350    1989-00398    October 28, 1989 00:00      Brevard     Florida   
1375      09-22941      April 18, 1973 14:26     Anderson      Kansas   
1379     090027656       July 24, 1974 09:51    Dickinson      Kansas   
1380      09-47152     August 17, 1979 14:29       Thomas      Kansas   
1381      09-51030       June 13, 1980 14:42       Thomas      Kansas   
1382    0900-66290   February 24, 1983 14:52     Doniphan      Kansas   
1385        501467   February 10, 2005 00:00   Sacramento  California   
1387       02-3947     August 18, 2002 00:00   Sacramento  California   
1388       02-0846   February 18, 2002 00:00   Sacramento  California   
1389         20827   February 10, 2002 00:00   Sacramento  California   
1393        995119   December 04, 1999 00:00   Sacramento  California   
1394        971812      April 26, 1997 00:00   Sacramento  California   
1395        943508  September 07, 1994 00:00   Sacramento  California   
1396        942187       June 03, 1994 00:00   Sacramento  California   
1397        933835    October 29, 1993 00:00   Sacramento  California   
1398        933194  September 10, 1993 00:00   Sacramento  California   
1400        932412       July 11, 1993 00:00   Sacramento  California   
1401        931388      April 20, 1993 00:00   Sacramento  California   
1411        882543     August 20, 1988 00:00   Sacramento  California   
1415        872068       July 26, 1987 00:00   Sacramento  California   
1438        87-147  September 21, 1987 11:48       Marion      Kansas   
1439    0900-92606    October 24, 1987 12:02    Dickinson      Kansas   
1440         89649      April 02, 1989 12:14        Geary      Kansas   
1441       92-1432   February 22, 1992 12:41      Bourbon      Kansas   
1442       95CR009   February 19, 1995 12:47        Trego      Kansas   
1443       0331739        May 14, 2004 14:37         Ford      Kansas   
1444       83-0729       June 23, 1983 15:00  Leavenworth      Kansas   

                                      recognizable_face  \
864                                                       
873                                                       
875      Not recognizable - Partial skeletal parts only   
903   Not recognizable - Partial remains with s

* **Index 1135 seems to be a 1-star**
* **Index 1438 Not recognizble, Estimated height and weight**

In [24]:
zeroStars[100:150]

case_number                date_found       county       state  \
1445     440010252      April 26, 1984 15:05    Wyandotte      Kansas   
1446        W62795     August 29, 1986 09:08      Wichita      Kansas   
1447      19004627  September 15, 1990 10:27      Johnson      Kansas   
1448        921073       June 25, 1992 10:37  Leavenworth      Kansas   
1449     94S101171    October 29, 1994 10:49     Sedgwick      Kansas   
1450         95267  September 21, 1995 11:00       Marion      Kansas   
1569      07-02351    October 19, 2007 15:51    San Diego  California   
1579      06-01785     August 24, 2006 19:40    San Diego  California   
1582   01-C050.NOV   November 12, 2001 14:21         Kent    Michigan   
1747      81201103      April 29, 2008 00:00       Travis       Texas   
1759     700018404     August 27, 2007 00:00    Fort Bend       Texas   
1761     07-180184       June 29, 2007 00:00      El Paso       Texas   
1763       07-0337        May 22, 2007 00:00      El Paso       Texas   
1764     200703644        May 14, 2007 00:00  Los Angeles  California   
1766       7040843      April 13, 2007 00:00      Cameron       Texas   
1776        703424      March 28, 1970 00:00  Los Angeles  California   
1846     200409674   December 21, 2004 00:00  Los Angeles  California   
1848     200407197  September 22, 2004 00:00  Los Angeles  California   
1852     200404291       June 04, 2004 00:00  Los Angeles  California   
1854     200402264      March 19, 2004 00:00  Los Angeles  California   
1856     200309997   December 31, 2003 00:00  Los Angeles  California   
1859     200308618   November 16, 2003 00:00  Los Angeles  California   
1866     200308083    October 26, 2003 00:00  Los Angeles  California   
1867     200307158  September 21, 2003 00:00  Los Angeles  California   
1870     200300979   February 04, 2003 00:00  Los Angeles  California   
1871     200300112    January 04, 2003 00:00  Los Angeles  California   
1872     200300134    January 04, 2003 00:00  Los Angeles  California   
1874     200209265   December 10, 2002 00:00  Los Angeles  California   
1875     200209265   December 10, 2002 00:00  Los Angeles  California   
1876     200209263   December 06, 2002 00:00  Los Angeles  California   
1877     200208637   November 18, 2002 00:00  Los Angeles  California   
1882        702670      April 28, 2007 00:00     Chambers       Texas   
1883     S07-06665      April 15, 2007 00:00     Victoria       Texas   
1884       07-0200      March 21, 2007 00:00      El Paso       Texas   
1887      06-08021      March 08, 2006 00:00      Hidalgo       Texas   
1890      05-30166  September 19, 2005 00:00      Hidalgo       Texas   
1894  05-06-14-032       June 14, 2005 00:00       Brooks       Texas   
1895     05-035354       June 10, 2005 00:00       Nueces       Texas   
1896      05-14410        May 10, 2005 00:00      Hidalgo       Texas   
1897      51281217        May 08, 2005 00:00       Travis       Texas   
1900      05-10397      April 06, 2005 00:00      Hidalgo       Texas   
1901        510266      April 05, 2005 00:00      Hidalgo       Texas   
1904      04-31669  September 26, 2004 00:00         Webb       Texas   
1906      04-17673       July 06, 2004 00:00      Hidalgo       Texas   
1907      04-22304       July 06, 2004 00:00      Hidalgo       Texas   
1908    2004061013       June 15, 2004 00:00       Zapata       Texas   
1909      04-14545       June 05, 2004 00:00      Hidalgo       Texas   
1910     2004-0799      April 16, 2004 00:00        Starr       Texas   
1911      04-06405      March 14, 2004 00:00      Hidalgo       Texas   
1913     2004-8407   February 27, 2004 00:00    Galveston       Texas   

                                      recognizable_face  \
1445                                                      
1446                                                      
1447                                                      
1448                                           

* **Index 1582 appears to be a 1-star**
* **Index 1747 Not recognizable, Estimated height and weight**

In [25]:
zeroStars[150:200]

case_number                date_found       county       state  \
1914             4018946   February 19, 2004 00:00       Dallas       Texas   
1915         2004-A00317    January 13, 2004 00:00      El Paso       Texas   
1917             3324247   November 20, 2003 00:00      El Paso       Texas   
1919            03-25504    October 21, 2003 00:00      Hidalgo       Texas   
1920       2003-00047360    October 07, 2003 00:00    Galveston       Texas   
1921            03-15294       June 25, 2003 00:00      Hidalgo       Texas   
1922            03-12512        May 23, 2003 00:00      Hidalgo       Texas   
1923            03-11658        May 17, 2003 00:00      Hidalgo       Texas   
1924           03-007976      April 08, 2003 00:00      Hidalgo       Texas   
1928            2002-239    October 13, 2002 00:00       Parmer       Texas   
1929             2029206    October 05, 2002 00:00      Hidalgo       Texas   
1931             2244120  September 01, 2002 00:00      El Paso       Texas   
1934          01B-014-02        May 21, 2002 00:00        Gregg       Texas   
1937             2002294      April 18, 2002 00:00      Hidalgo       Texas   
1939             2075102      March 16, 2002 00:00      El Paso       Texas   
1942             02 2660    January 31, 2002 00:00      Hidalgo       Texas   
1944           200208348   November 07, 2002 00:00  Los Angeles  California   
1946           200205130       July 09, 2002 00:00  Los Angeles  California   
1947           200204962       July 02, 2002 00:00  Los Angeles  California   
1949           200204857       June 28, 2002 00:00  Los Angeles  California   
1951           200303606        May 05, 2002 00:00  Los Angeles  California   
1952           200203166   February 23, 2002 00:00  Los Angeles  California   
1953           200201310   February 14, 2002 00:00  Los Angeles  California   
1987           200002569      April 06, 2000 00:00  Los Angeles  California   
1999             9907845   November 17, 1999 00:00  Los Angeles  California   
2000             9906944    October 15, 1999 00:00  Los Angeles  California   
2002             9906536  September 28, 1999 00:00  Los Angeles  California   
2003             9906330  September 20, 1999 00:00  Los Angeles  California   
2004             9906057  September 07, 1999 00:00  Los Angeles  California   
2008             9904460       June 28, 1999 00:00  Los Angeles  California   
2009             9905266       June 14, 1999 00:00  Los Angeles  California   
2010             9904459       June 13, 1999 00:00  Los Angeles  California   
2011             9903407        May 07, 1999 00:00  Los Angeles  California   
2012             9900671    January 24, 1999 00:00  Los Angeles  California   
2016           200205053    October 23, 1998 00:00  Los Angeles  California   
2039                1256     August 28, 2001 00:00     Presidio       Texas   
2040             0125-01     August 15, 2001 00:00    Val Verde       Texas   
2045             1180103       June 29, 2001 00:00      El Paso       Texas   
2046            01-11228        May 29, 2001 00:00      Hidalgo       Texas   
2071   DALLAS PD 759192J  September 27, 2000 00:00       Dallas       Texas   
2076             607631J     August 05, 2000 00:00       Dallas       Texas   
2077               10624       July 09, 2000 00:00        Gregg       Texas   
2078  DALLAS PD 477310-J       June 23, 2000 00:00       Dallas       Texas   
2080          000612-003       June 12, 2000 00:00      Edwards       Texas   
2081               62733       June 01, 2000 00:00       Kenedy       Texas   
2082            00-07265        May 08, 2000 00:00      Hidalgo       Texas   
2089             1137-00      April 01, 2000 00:00       Dallas       Texas   
2092              291-00      March 15, 2000 00:00         Frio       Texas   
2093                 501    January 10, 2000 00:00        Gregg       Texas   
2099            99288321    October 15, 1999 00:00      El Paso

* **Index 1914 Not recognizable, Estimated height and weight**
* **Index 1937 Not recognizable, Estimated height and weight**
* **Index 2039 Not recognizable, Estimated height and weight**

In [26]:
zeroStars[200:250]

case_number                date_found       county        state  \
2107    99-13016     August 04, 1999 00:00      Hidalgo        Texas   
2108    99-12925     August 01, 1999 00:00      Hidalgo        Texas   
2111    99-10524       June 21, 1999 00:00      Hidalgo        Texas   
2115    99-09653       June 06, 1999 00:00      Hidalgo        Texas   
2123    98-23923   November 18, 1998 00:00      Hidalgo        Texas   
2129    98005278    October 03, 1998 00:00    Van Zandt        Texas   
2130    98025246  September 28, 1998 00:00    Jefferson        Texas   
2136    98071961       July 14, 1998 00:00       Kenedy        Texas   
2137     9507848    October 06, 1995 00:00  Los Angeles   California   
2159   CC-322-98      April 02, 1998 00:00       Collin        Texas   
2165     9502678      April 07, 1995 00:00  Los Angeles   California   
2170     899954F   November 02, 1997 00:00       Dallas        Texas   
2178     9710823       July 26, 1997 00:00      Cameron        Texas   
2182    71-12645   December 02, 1971 00:00  Los Angeles   California   
2184     97-2715       July 05, 1997 00:00    Galveston        Texas   
2185   971850939       July 04, 1997 00:00       Travis        Texas   
2186    97042191        May 29, 1997 00:00      Cameron        Texas   
2188    97031556      March 31, 1997 00:00       Kenedy        Texas   
2193     9905837   November 14, 1994 00:00  Los Angeles   California   
2200   96-121458   December 09, 1996 00:00       Kenedy        Texas   
2201     9407793  September 11, 1994 00:00  Los Angeles   California   
2206    96271121  September 27, 1996 00:00      El Paso        Texas   
2207    96238313     August 25, 1996 00:00      El Paso        Texas   
2209    96071349       July 27, 1996 00:00       Kenedy        Texas   
2210    96-10958       July 18, 1996 00:00        Ector        Texas   
2213    10956031       June 11, 1996 00:00         Kerr        Texas   
2214     9604265        May 10, 1996 00:00      Johnson        Texas   
2219     9604068      April 16, 1996 00:00      Cameron        Texas   
2224   200208752    January 01, 1994 00:00  Los Angeles   California   
2234        5979     August 15, 1995 00:00       Uvalde        Texas   
2236    95226052     August 14, 1995 00:00      El Paso        Texas   
2237    95071068       July 28, 1995 00:00       Kenedy        Texas   
2239     467659D       June 06, 1995 00:00       Dallas        Texas   
2241   95-F04742        May 19, 1995 00:00      El Paso        Texas   
2242     379410D        May 11, 1995 00:00       Dallas        Texas   
2254   94-079050   December 26, 1994 00:00       Nueces        Texas   
2262     94-2449       July 30, 1994 00:00        Starr        Texas   
2272   65852394L       June 21, 1994 00:00       Harris        Texas   
2283    94-01362    January 23, 1994 00:00   Montgomery        Texas   
2284   001001-94    January 20, 1994 00:00        Gregg        Texas   
2286    50310-93   December 20, 1993 00:00    Jefferson        Texas   
2288    93-68701    October 30, 1993 00:00         Webb        Texas   
2290     9320263       July 11, 1993 00:00       Dallas        Texas   
2291    H06-8261        May 23, 2006 00:00      Hancock  Mississippi   
2293     541415B       July 08, 1993 00:00       Dallas        Texas   
2295    93107344      April 17, 1993 00:00      El Paso        Texas   
2299      H93001    January 30, 1993 00:00         Hill        Texas   
2308    91-09412    October 12, 1991 00:00  Los Angeles   California   
2311    91-09115    October 03, 1991 00:00  Los Angeles   California   
2317     9107078     August 03, 1991 00:00  Los Angeles   California   

                                   recognizable_face                    race  \
2107                                                                   White   
2108                                                                   White   
2111                                                                   White   
2115        

* **Index 2288 Not recognizable, Estimated weight and height**
* **Index 2290 Not recognizable, Estimated weight and height**
* **Index 2291 Recog face, Estimated weight and height**

In [27]:
zeroStars[250:300]

case_number                date_found       county       state  \
2319      9212599        May 21, 1992 00:00    Fort Bend       Texas   
2321     92-09651        May 08, 1992 00:00      Hidalgo       Texas   
2322      9106650       July 21, 1991 00:00  Los Angeles  California   
2331     92007557      March 18, 1992 00:00       Dallas       Texas   
2346    199012002   December 10, 1990 00:00  Los Angeles  California   
2356      9008914  September 10, 1990 00:00  Los Angeles  California   
2359    91-01-333   December 18, 1991 00:00       Nueces       Texas   
2386        30275      March 26, 1991 00:00     Victoria       Texas   
2387      218886Z      March 16, 1991 00:00       Dallas       Texas   
2397     89-10982   November 13, 1989 00:00  Los Angeles  California   
2415     89-08624  September 05, 1989 00:00  Los Angeles  California   
2419      431011Y       June 17, 1990 00:00       Dallas       Texas   
2422      345129Y        May 18, 1990 00:00       Dallas       Texas   
2424     89-07404       July 30, 1989 00:00  Los Angeles  California   
2429      211061Y      March 28, 1990 00:00       Dallas       Texas   
2449      8900608    January 15, 1989 00:00  Los Angeles  California   
2450      791287X   November 08, 1989 00:00       Dallas       Texas   
2462    89-09-276  September 25, 1989 00:00   Williamson       Texas   
2463      892091T       June 15, 1989 00:00      Tarrant       Texas   
2467       L67564        May 27, 1989 00:00      Liberty       Texas   
2477      8903717      April 26, 1989 00:00      Johnson       Texas   
2485      88-0029   December 28, 1988 00:00       Dallas       Texas   
2486    CC-605-88   December 15, 1988 00:00       Collin       Texas   
2487      787963W   December 05, 1988 00:00       Dallas       Texas   
2491      445390W       July 19, 1988 00:00       Dallas       Texas   
2493      88-0627       June 17, 1988 00:00        Starr       Texas   
2496      296622W        May 21, 1988 00:00       Dallas       Texas   
2500      141170W      March 13, 1988 00:00       Dallas       Texas   
2506  1987R054784    October 03, 1987 00:00       Dallas       Texas   
2515      8800608    January 15, 1988 00:00  Los Angeles  California   
2519      8712642   December 29, 1987 00:00  Los Angeles  California   
2525      8710251    October 25, 1987 00:00  Los Angeles  California   
2526      8709160  September 23, 1987 00:00  Los Angeles  California   
2527      8708711  September 09, 1987 00:00  Los Angeles  California   
2531      8706646       July 10, 1987 00:00  Los Angeles  California   
2543      8700782    January 19, 1987 00:00  Los Angeles  California   
2546      8616332   December 13, 1986 00:00  Los Angeles  California   
2550     87-15359  September 13, 1987 00:00       Taylor       Texas   
2563      8610986     August 23, 1986 00:00  Los Angeles  California   
2564      8610425     August 11, 1986 00:00  Los Angeles  California   
2566      358595U       June 20, 1987 00:00       Dallas       Texas   
2569       L55066       June 14, 1987 00:00      Liberty       Texas   
2585      205244U      April 12, 1987 00:00       Dallas       Texas   
2590      87-0463      April 03, 1987 00:00    Guadalupe       Texas   
2608        85776    January 15, 1985 00:00  Los Angeles  California   
2620      W870017    January 07, 1987 00:00     Chambers       Texas   
2625       845513      April 29, 1984 00:00  Los Angeles  California   
2628      618382T    October 11, 1986 00:00       Dallas       Texas   
2629      515286T     August 27, 1986 00:00       Dallas       Texas   
2641       842245   February 17, 1984 00:00  Los Angeles  California   

                                      recognizable_face  \
2319     Not recognizable - Partial skeletal parts only   
2321     Not recognizable - Partial skeletal parts only   
2322                                                      
2331                                                      
2346                                  

* **Index 2321 Not recognizable, Cannot Estimate, Estimated**
* **Index 2493 Not recognizable, Estimated weight and height (sex Unsure)

In [28]:
zeroStars[300:350]

case_number                date_found       county       state  \
2642       84243    January 06, 1984 00:00  Los Angeles  California   
2651     8311406  September 11, 1983 00:00  Los Angeles  California   
2654      838600       July 11, 1983 00:00  Los Angeles  California   
2664      834051      March 30, 1983 00:00  Los Angeles  California   
2681      828329       July 03, 1982 00:00  Los Angeles  California   
2699     8115184   November 29, 1981 00:00  Los Angeles  California   
2700     8114969   November 24, 1981 00:00  Los Angeles  California   
2702     8114268   November 08, 1981 00:00  Los Angeles  California   
2704     8113382    October 20, 1981 00:00  Los Angeles  California   
2705     8113304    October 19, 1981 00:00  Los Angeles  California   
2706    81-13259    October 18, 1981 00:00  Los Angeles  California   
2711     8110511     August 16, 1981 00:00  Los Angeles  California   
2712      819955     August 03, 1981 00:00  Los Angeles  California   
2714      819289       July 19, 1981 00:00  Los Angeles  California   
2716      818692       July 06, 1981 00:00  Los Angeles  California   
2717      817322       June 07, 1981 00:00  Los Angeles  California   
2718      815692        May 02, 1981 00:00  Los Angeles  California   
2725      812852   February 28, 1981 00:00  Los Angeles  California   
2728      812565   February 21, 1981 00:00  Los Angeles  California   
2732       81473    January 10, 1981 00:00  Los Angeles  California   
2755      804915      April 14, 1980 00:00  Los Angeles  California   
2757      803808      March 21, 1980 00:00  Los Angeles  California   
2761     8002317   February 17, 1980 00:00  Los Angeles  California   
2763      801260    January 25, 1980 00:00  Los Angeles  California   
2766     7915967   December 22, 1979 00:00  Los Angeles  California   
2776     7907440       June 11, 1979 00:00  Los Angeles  California   
2782     7903152      March 10, 1979 00:00  Los Angeles  California   
2788     7810909  September 04, 1978 00:00  Los Angeles  California   
2789   197810883  September 03, 1978 00:00  Los Angeles  California   
2792     7807811       June 16, 1978 00:00  Los Angeles  California   
2795      784826      April 14, 1978 00:00  Los Angeles  California   
2796     7804407      April 02, 1978 00:00  Los Angeles  California   
2797      783980      March 25, 1978 00:00  Los Angeles  California   
2798     7803500      March 15, 1978 00:00  Los Angeles  California   
2800     7712406    October 14, 1977 00:00  Los Angeles  California   
2815     7606288        May 19, 1976 00:00  Los Angeles  California   
2827      736512      April 14, 1973 00:00  Los Angeles  California   
2829     263325T        May 10, 1986 00:00       Dallas       Texas   
2830     143384T      March 15, 1986 00:00       Dallas       Texas   
2833     233605T      April 26, 1986 00:00       Dallas       Texas   
2836   CC-010-95      March 14, 1986 00:00       Collin       Texas   
2839      21134T    January 12, 1986 00:00       Dallas       Texas   
2847       19283  September 08, 1985 00:00     Victoria       Texas   
2849    85-18123       July 08, 1985 00:00       Travis       Texas   
2857     1850190    January 19, 1985 00:00     McLennan       Texas   
2865   841029041    October 29, 1984 00:00       Nueces       Texas   
2867  84-10-5632    October 02, 1984 00:00    Freestone       Texas   
2868     842785T    October 01, 1984 00:00      Tarrant       Texas   
2869      84-287  September 12, 1984 00:00     Live Oak       Texas   
2889   73-01-012   December 28, 1973 00:00       Nueces       Texas   

                                      recognizable_face  \
2642                                                      
2651     Not recognizable - Partial skeletal parts only   
2654                                                      
2664                                                      
2681                                                      
2699                         

* **Index 2651 Not recognizable, Estimated weight and height**
* **Index 2757 Not recognizable, Measured weight, Estimated height**

In [29]:
zeroStars[350:400]

case_number                date_found         county        state  \
2899       76-2007       June 20, 1976 00:00         Travis        Texas   
2905        772496       June 03, 1977 00:00         Dallas        Texas   
2912       0507810       June 15, 1983 00:00         Denton        Texas   
2918       8305503      April 25, 1983 00:00         Denton        Texas   
2921     ME82-1050   November 29, 1982 00:00         Travis        Texas   
2922       822247T  September 12, 1982 00:00        Tarrant        Texas   
2924     82-01-018       July 23, 1982 00:00         Nueces        Texas   
2927          8143   December 21, 1981 00:00         Oldham        Texas   
2932      TA816903  September 23, 1981 00:00         Nueces        Texas   
2934       BREN001     August 28, 1981 00:00        Trinity        Texas   
2944     80-01-004     August 04, 1980 00:00         Nueces        Texas   
2950         D1843      April 10, 1980 00:00      Galveston        Texas   
2952      37408-79   December 19, 1979 00:00      Jefferson        Texas   
2974  2003-6003515       July 30, 2003 00:00        Chatham      Georgia   
2980      11980189   December 16, 1988 00:00           Dade      Georgia   
2981     501850188   February 14, 1988 00:00        Jenkins      Georgia   
2993      00-M1003      March 23, 2000 00:00          Adams  Mississippi   
2995     99-032875   February 19, 1999 00:00          Adams  Mississippi   
3000       980211A   February 01, 1998 16:19     Tuscarawas         Ohio   
3001        971012       June 14, 1997 00:00          Adams  Mississippi   
3003     SPD96-062        May 17, 1996 15:34           Erie         Ohio   
3013        296730  September 03, 1989 00:00           Dane    Wisconsin   
3015       89-7559      April 04, 1989 12:27         Lorain         Ohio   
3034         31624      April 18, 1967 00:00         Ramsey    Minnesota   
3035         31285   November 02, 1966 00:00         Ramsey    Minnesota   
3044     1964-1441    October 05, 1964 00:00       Hennepin    Minnesota   
3046     1965-0712        May 28, 1965 00:00       Hennepin    Minnesota   
3064       45-7832   December 25, 1977 00:00          Floyd      Indiana   
3065     88-033-M2      April 14, 1988 00:00       La Porte      Indiana   
3066       72-0781   February 22, 1984 00:00          Union      Indiana   
3067      12887742   December 10, 1988 00:00         Monroe      Indiana   
3068      41-10436      April 22, 1990 00:00         Monroe      Indiana   
3069        851635       June 09, 1985 00:00         Monroe      Indiana   
3070      99-14848      April 20, 1999 00:00         Monroe      Indiana   
3072     J08005902      April 27, 2008 00:00         Monroe      Indiana   
3074       C941286    October 18, 1994 00:00        Hancock      Indiana   
3123      91-01866  September 08, 1991 00:00      San Diego   California   
3142      91-01982  September 24, 1991 00:00      San Diego   California   
3153     BMA040108      March 03, 1987 00:00          Macon     Missouri   
3318       ME78-79   February 13, 1979 00:00        Alachua      Florida   
3756     1987-0091  September 18, 1987 00:00  San Francisco   California   
4026      11929-77   December 17, 1977 16:00         Placer   California   
4273   PAT-0626-10   February 10, 2010 21:31            All  Puerto Rico   
4327     03-020257    October 02, 2003 09:01          Henry      Indiana   
4370     1971-1063      April 26, 1971 10:30     Miami-Dade      Florida   
4379       02-0116    January 24, 2002 10:13       Honolulu       Hawaii   
4547     ML96-7381    October 20, 1996 14:24         Harris        Texas   
4591     1979-7826        May 21, 1979 07:41         Denver     Colorado   
4592    1980-80248      April 25, 1980 08:18         Denver     Colorado   
4593    1981-82214      March 21, 1981 08:36         Denver     Colorado   

                                      recognizable_face  \
2899                                                      
2905        

* **Index 2922 Not recognizable, Estimated weight and height**
* **Index 2950 Not recognizable, Estimated weight and height**
* **Index 2995 Not recognizable, Estimated weight and height**
* **Index 3000 Not recognizable, Estimated weight and height**
* **Index 3001 Not recognizable, Estimated weight and height**
* **Index 3003 Not recognizable, Estimated weight, Measured height**
* **Index 3013 Not recognizable, Cannot Estimate weight, Estimated height**
* **Index 3074 Not recognizable, Cannot Estimate weight, Estimated height**
* **Index 3153 Not recognizable, Estimated weight and height**
* **Index 3318 Not recognizable, Cannot Estimate weight, Estimated height**
* **Index 4026 Recognizable face, Measured weight, Measured height -> SHOULD BE A 1-STAR?**
* **Index 4273 Recognizable face, Measured weight, Measured height -> SHOULD BE A 1-STAR?**


In [30]:
zeroStars[400:450]

case_number                date_found        county         state  \
4595      1983-00000    October 16, 1983 08:46        Denver      Colorado   
4597       1987-0000       June 07, 1987 09:13        Denver      Colorado   
4598      1987-00000    January 30, 1987 09:20        Denver      Colorado   
4599       1988-1604        May 02, 1988 09:40        Denver      Colorado   
4600       1996-0000    January 26, 1996 09:54        Denver      Colorado   
4601       1997-0106    January 08, 1997 10:04        Denver      Colorado   
4602       1999-0000   February 25, 1999 10:14        Denver      Colorado   
4603       2002-0849   February 02, 2002 10:30        Denver      Colorado   
4604       2005-2587       June 25, 2005 10:35        Denver      Colorado   
4793     7929-126-91      April 14, 1991 14:33          Yolo    California   
4861  2004-10-FA-564   December 07, 2004 12:50        Hardee       Florida   
5280        10-06799   November 08, 2010 11:44  Hillsborough       Florida   
79           01-0577      April 25, 2001 00:00       Spokane    Washington   
84           91-1777  September 18, 1991 13:47        Fulton       Georgia   
95           85-0058    January 13, 1985 11:01    Palm Beach       Florida   
96           85-1000    October 10, 1985 11:39    Palm Beach       Florida   
113       2002-00573       June 12, 2002 00:00    Palm Beach       Florida   
114       1999-00704   December 21, 1999 00:00        Glades       Florida   
115       1996-00508        May 23, 1996 00:00    Palm Beach       Florida   
129       1987-00611      March 02, 1987 00:00    Miami-Dade       Florida   
132       1986-01067    October 21, 1986 00:00        Nassau       Florida   
141       1986-00097   February 17, 1986 00:00        Glades       Florida   
143       1985-00000   December 16, 1985 00:00       Brevard       Florida   
153       1984-01118   December 02, 1984 00:00    Palm Beach       Florida   
190       1980-00480   December 14, 1980 00:00           Lee       Florida   
191       1980-00637   December 04, 1980 00:00        Putnam       Florida   
206       1979-00805     August 26, 1979 00:00         Duval       Florida   
211       1979-00000      March 03, 1979 00:00    Palm Beach       Florida   
214       1978-00000   February 27, 1978 00:00    Palm Beach       Florida   
230       1973-00267        May 21, 1973 00:00           Lee       Florida   
240        2001-1053   November 24, 2001 00:00    Palm Beach       Florida   
243       1996-00554       June 12, 1996 00:00    Palm Beach       Florida   
309         90-20470       July 16, 1990 00:00       Passaic    New Jersey   
331          01-3586   November 10, 2001 00:00      Maricopa       Arizona   
332          01-3585   November 10, 2001 00:00      Maricopa       Arizona   
339          05-3857    October 10, 2005 00:00      Maricopa       Arizona   
345          03-4051   December 01, 2003 00:00      Maricopa       Arizona   
348          A91-499    January 27, 1991 12:41      Columbia       Georgia   
353         A91-3459    October 31, 1991 00:00      Richmond       Georgia   
354          M89-418    January 25, 1989 00:00     Dougherty       Georgia   
355        M91-01113   February 27, 1991 10:36          Cook       Georgia   
370           08-757   February 21, 2008 00:00  Philadelphia  Pennsylvania   
371         81-26830    October 20, 1981 10:15       Forsyth       Georgia   
397            32301        May 10, 2003 00:00    Sacramento    California   
398           942225       June 07, 1994 00:00    Sacramento    California   
401           883519   November 13, 1988 00:00    Sacramento    California   
403         12013382    January 29, 1992 10:42     Wyandotte        Kansas   
455       None given   February 01, 1992 16:12          Kent      Michigan   
456       None given    January 02, 1992 16:33          Kent      Michigan   
462      99-C006.DEC   December 03, 1999 09:24          Kent      Michigan   

                              

In [31]:
zeroStars[450:500]

case_number                date_found       county       state  \
463        00-L053.AUG     August 30, 2000 11:07         Kent    Michigan   
489            08-0786     August 26, 2008 00:00      Willacy       Texas   
490           08-16515       June 12, 2008 00:00       Travis       Texas   
493     DCS0 07-126688   December 19, 2007 00:00       Dallas       Texas   
497             738652        May 13, 1971 00:00  Los Angeles  California   
515          200309962   December 30, 2003 00:00  Los Angeles  California   
528         2006030514      March 29, 2006 00:00      Kleberg       Texas   
531         2006-03769    January 26, 2006 00:00    Galveston       Texas   
533         2006010061    January 03, 2006 00:00    Matagorda       Texas   
534            3853-05   November 23, 2005 00:00       Dallas       Texas   
540           04-62939   December 23, 2004 00:00        Smith       Texas   
543            4060638       June 14, 2004 00:00       Kenedy       Texas   
544            1014-04   February 05, 2004 00:00     Presidio       Texas   
545          03-121433   December 18, 2003 00:00       Kenedy       Texas   
546       EPME 03-2131    October 27, 2003 00:00      El Paso       Texas   
547           70427-03    October 10, 2003 00:00    Jefferson       Texas   
549          02A000645    January 12, 2002 00:00   Montgomery       Texas   
550           Jul-1979    January 03, 2002 00:00      Hidalgo       Texas   
563         011226-001   December 26, 2001 00:00      Edwards       Texas   
564         01/12/1968       July 16, 2001 00:00       Kenedy       Texas   
566  DALLAS PD 529305K       July 08, 2001 00:00       Dallas       Texas   
567           Aug-1968      March 28, 2001 00:00      Hidalgo       Texas   
568         01-02-451A   February 25, 2001 00:00       Kenedy       Texas   
569           01-03452   February 06, 2001 00:00      Hidalgo       Texas   
572          120012581   December 23, 2000 00:00        Gregg       Texas   
576  DALLAS PD 748303J  September 23, 2000 00:00       Dallas       Texas   
578            001994T      April 23, 2000 00:00      Tarrant       Texas   
580           99-12244   December 06, 1999 00:00      Liberty       Texas   
584           99-12707       July 29, 1999 00:00      Hidalgo       Texas   
585           99032776       June 15, 1999 00:00       Nueces       Texas   
587              91515    January 15, 1999 00:00        Ector       Texas   
589          98A011288     August 06, 1998 00:00   Montgomery       Texas   
590          98A008113       June 11, 1998 00:00   Montgomery       Texas   
591          98-051892        May 04, 1998 00:00       Kenedy       Texas   
594          97A012812  September 30, 1997 00:00   Montgomery       Texas   
595          97-09-072  September 23, 1997 00:00       Burnet       Texas   
596             974509  September 07, 1997 00:00     Atascosa       Texas   
598           B97-1003   February 17, 1997 00:00    Jefferson       Texas   
599          CC-069-97    October 28, 2011 11:11       Collin       Texas   
600             960572   December 02, 1996 00:00    Tom Green       Texas   
601           96323181   November 18, 1996 00:00      El Paso       Texas   
604              96140      April 25, 1996 00:00         Leon       Texas   
610          95-039437        May 16, 1995 00:00      Cameron       Texas   
611           95042189      April 13, 1995 00:00         Polk       Texas   
612          95-02-951    January 28, 1995 00:00       Kenedy       Texas   
614              10352     August 14, 1994 00:00        Falls       Texas   
619            9109634    October 11, 1991 00:00  Los Angeles  California   
622               7084   February 17, 1992 00:00     Presidio       Texas   
627          90-03244A      March 03, 1990 00:00       Kenedy       Texas   
632             892846      March 27, 1989 00:00      Cameron       Texas   

                                     recognizable_face  \
463  Not recognizable - P

In [32]:
zeroStars[500:]

case_number                date_found         county        state  \
670        7815754   December 14, 1978 00:00    Los Angeles   California   
672        252043T        May 05, 1986 00:00         Dallas        Texas   
673       85-28903   November 29, 1985 00:00     Montgomery        Texas   
676        85-1454     August 08, 1985 00:00         Burnet        Texas   
678          84234   November 17, 1984 00:00            Lee        Texas   
684       51360-83    January 27, 1983 00:00         Harris        Texas   
689        3110580    October 01, 1981 00:00       Brazoria        Texas   
692      2002-0523  September 04, 2002 00:00          Perry  Mississippi   
696      123090180      April 03, 1980 00:00        Candler      Georgia   
698        2075879       July 03, 2002 00:00           Dane    Wisconsin   
702        93-594T      March 25, 1993 14:45         Geauga         Ohio   
704          30526   December 05, 1965 00:00         Ramsey    Minnesota   
706          30872      April 24, 1966 00:00         Ramsey    Minnesota   
707          39897       July 09, 1977 00:00         Ramsey    Minnesota   
708       1965-529      April 19, 1965 00:00       Hennepin    Minnesota   
713       02-21599   November 24, 2002 00:00          Floyd      Indiana   
865      1975-0435      March 07, 1975 00:00  San Francisco   California   
869      1981-1613   November 27, 1981 00:00  San Francisco   California   
873      1984-0648        May 09, 1984 00:00  San Francisco   California   
985        78-0197   February 22, 1978 15:42       Honolulu       Hawaii   
1023     PA-95-234      April 13, 1995 15:34           Polk        Texas   
1141  Not Reported    January 17, 2000 09:41         Vernon     Missouri   
1143     2006-2326  September 15, 2006 13:40     Miami-Dade      Florida   
1149   pat-1878-10      April 30, 2010 10:55            All  Puerto Rico   
1538                                                                       
2246                     June 01, 1977 00:00     St. Mary's     Maryland   

                                      recognizable_face  \
670      Not recognizable - Partial skeletal parts only   
672                                                       
673   Not recognizable - Partial remains with soft t...   
676                                                       
678   Not recognizable - Partial remains with soft t...   
684   Not recognizable - Partial remains with soft t...   
689   Not recognizable - Partial remains with soft t...   
692      Not recognizable - Partial skeletal parts only   
696                                                       
698         Not recognizable - Decomposing/putrefaction   
702         Not recognizable - Decomposing/putrefaction   
704                                                       
706                                                       
707                                                       
708                                                       
713                                                       
865                                                       
869                                                       
873                                                       
985   Not recognizable - Near complete or complete s...   
1023     Not recognizable - Partial skeletal parts only   
1141  Not recognizable - Near complete or complete s...   
1143                                  Recognizable face   
1149  Not recognizable - Near complete or complete s...   
1538                                                      
2246                                  Recognizable face   

                        race     sex                        weight  \
670                    White    Male        ,\n\t\t\t\t\tEstimated   
672                    White    Male     112,\n\t\t\t\t\tEstimated   
673                   Unsure  Unsure        ,\n\t\t\t\t\tEstimated   
676                    White    Male     155,\n\t\t\t\t\tEstimated   
678       